apps/server/src/app.module.ts

```diff
@@ -24,8 +27,21 @@ import { ConfigModule } from '@nestjs/config';
     ConfigModule.forRoot({
       envFilePath: '.env',
     }),
+    AuthModule,
+    PassportModule,
+    JwtModule.register({
+      secret: process.env.JWT_SECRET,
```

Comment:
What does secret mean?

Reply:
it's what's used to generate the jwt token and decode it